source/collections/time-series-collection.txt

```diff
@@ -47,23 +47,41 @@ Procedure
            - Description 
 
          * - ``metaField``
-           - The name of the field which contains metadata in each time series 
+           - The name of the field that contains metadata in each time series 
              document. The metadata in the specified field should be data that 
              is used to label a unique series of documents. 
 
          * - ``granularity``
-           - The ``granularity`` field allows specifying a coarser granularity so 
-             measurements over a longer time span can be more efficiently stored 
-             and queried.
+           - Lets you specify a coarser granularity so measurements over a 
```

Comment:
[suggestion]

I feel like this description might read with more parallelism compared to the others if we remove "lets you" and instead say "Specifies a coarser granularity..."

Reply:
Ah my bad for not catching that the first time around. Updated!